site/docs/4.3/components/forms.md

```diff
@@ -23,7 +23,7 @@ Here's a quick example to demonstrate Bootstrap's form styles. Keep reading for
   </div>
   <div class="form-group">
     <label for="exampleInputPassword1">Password</label>
-    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Enter password">
```

Comment:
i'd remove this placeholder altogether, as well as the `placeholder="Enter email"` in the previous input